README.md

```diff
@@ -4,7 +4,26 @@
 [![Chat](https://img.shields.io/static/v1.svg?label=chat&message=mattermost&color=145dbf)](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind)
 
 # Codewind for VS Code
-Create and develop cloud-native, containerized web applications from VS Code.
+Codewind for VS Code provides tools to help build cloud-native, containerized applications from VS Code, regardless of which runtime or language you use. Rapidly create an application from a template project and launch, update, test, and debug in Docker containers on the desktop.
+
+## Why use Codewind?
+- Get started quickly with templates or samples.
+- Use the tools, runtimes, and languages that you're already familiar with.
+- Pull in existing applications and get them ready for the cloud.
+- Enjoy quick, iterative development and fast feedback.
+- Learn as you go.
+
+## Features
+- Create new containerized projects or add existing ones.
+- View your containerized projects, including the application and build status.
+- Debug Microprofile/Java EE, Node.js, and Spring applications.
+- View application and build logs in the Output view.
+- See validation errors in the Problems view.
+- Open a shell session into an application container.
+- Toggle the project auto build setting and manually initiate project builds.
+- Open your application or the application monitor in a browser.
```

Comment:
Maybe instead of Open we say "Launch"?

Reply:
Our word usage page says, "Do not use unless it is part of the terminology of the product that you are documenting. Use "start" or "open". I changed it to "start" if that's OK.